README.rst

```diff
@@ -28,13 +28,35 @@ degradation in accuracy. Although all major deep learning frameworks support FP1
 not available today.
 
 TE addresses the problem of FP8 support by providing APIs that integrate with popular Large Language
-Model (LLM) libraries. It provides python layer (initially supporting pyTorch, with support for more
-frameworks in the future) consisting of modules to easily build Transformer layer as well as
-framework agnostic library in C++ including structs and kernels needed for FP8 support. Modules
-provided by TE internally maintain scaling factors and other values needed for FP8 training, greatly
+Model (LLM) libraries. It provides python layer consisting of modules to easily build Transformer
+layer as well as framework agnostic library in C++ including structs and kernels needed for FP8 support.
+Modules provided by TE internally maintain scaling factors and other values needed for FP8 training, greatly
 simplifying for the users.
 
-Transformer Engine in action:
+Support Matrix:
+
++-----------+-----------+
+| Framework | Status    |
++===========+===========+
+|  pyTorch  | Supported |
++-----------+-----------+
+|    JAX    | Supported |
```

Comment:
Not sure how I feel about it - there are some feature gaps (on both sides actually, since e.g. JAX supports RMSNorm and GeGLU I believe while pyTorch does not have them exposed yet, while pyTorch has attention), so maybe that table should be more specific (or maybe not exist at all) - we should make sure that we define what "supported" means first.

Also, I recall hearing that currently the JAX integration requires NGC JAX container, is that still the case? Either way, we should specify which version of JAX we target.

Reply:
Make sense, remove it till we have definiation of `Support`.

Reply:
> Also, I recall hearing that currently the JAX integration requires NGC JAX container, is that still the case? Either way, we should specify which version of JAX we target.

Not know either, will ask internally

Reply:
> Also, I recall hearing that currently the JAX integration requires NGC JAX container, is that still the case? Either way, we should specify which version of JAX we target.

The TE/JAX does not use any APIs that only in NGC JAX container,, but it is good to run TE in NGC JAX container for the best performance.